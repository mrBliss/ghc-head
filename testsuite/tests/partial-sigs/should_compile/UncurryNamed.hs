{-# LANGUAGE PartialTypeSignatures, NamedWildcards #-}
module UncurryNamed where

unc :: (_a -> _b -> _c) -> (_a, _b) -> _c
unc = uncurry
