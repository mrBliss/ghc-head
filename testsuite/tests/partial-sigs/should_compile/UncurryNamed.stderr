TYPE SIGNATURES
  unc ::
    forall w_a w_b w_c.
    (w_a -> w_b -> w_c) -> (w_a, w_b) -> w_c
TYPE CONSTRUCTORS
COERCION AXIOMS
Dependent modules: []
Dependent packages: [base, ghc-prim, integer-gmp]
